index.html

```diff
@@ -38,11 +38,26 @@
 				</div>
 				<a class="header__menu-icon">
 					<object
+						class="header__menu-icon-svg"
```

Comment:
Don't use svg in your class name as this may change later

Reply:
Done